Clairctl [No config file used]

Hi jgsquare, first of all thank you for your amazing job.

I ve the following issue:
Clairctl doesn't see my config file (clairctl.yml) even in the default path $HOME/clairctl.yml.

Here are the specifications:

- I'm using  the Master branch version of clairctl on a Debian8

The problem:
```
root@debian:/home/superuser/stage/clairctl# clairctl health --config /root/clairctl.yml --log-level debug
2017-06-02 11:53:00.143418 D | config: No config file used
2017-06-02 11:53:00.143581 D | clair: requesting health on: http://localhost:6061/health

Clair: ✔
```
My config file:
```
root@debian:/home/superuser/stage/clairctl# ls $HOME/clairctl.yml
/root/clairctl.yml
```
```
clairctl:
  interface: br-5ae2491b4bb1
clair:
	port: 6060
	healthPort: 6061
	uri: http://172.22.0.3
report:
	path: ./reports
	format: html
docker:
	insecure-registries:
		- "my-own-registry:5000"
```
When i try to make a push  (even with the --config option) i've the following error
```
root@debian:/home/superuser/stage/clairctl# clairctl push -l debian --log-level debug
2017-06-02 12:07:03.500975 D | config: No config file used
2017-06-02 12:07:03.503096 I | config: retrieving interface for local IP
2017-06-02 12:07:03.503107 D | config: no interface provided, looking for docker0
2017-06-02 12:07:03.503631 D | server: Update local server port from "0" to "42121"
2017-06-02 12:07:03.503639 I | server: Starting Server on 172.17.0.1:42121
2017-06-02 12:07:03.517342 D | dockercli: docker image to save: debian:latest
2017-06-02 12:07:03.517357 D | dockercli: saving in: /tmp/clairctl/debian/blobs
2017-06-02 12:07:07.977692 I | config: retrieving interface for local IP
2017-06-02 12:07:07.977720 D | config: no interface provided, looking for docker0
2017-06-02 12:07:07.978190 I | clair: using http://172.17.0.1:42121/local as local url
2017-06-02 12:07:07.978202 I | clair: Pushing Layer 1/1 [de9cd0e84c73]
2017-06-02 12:07:08.043134 D | dockerdist: registry.LookupPushEndpoints error: Missing client certificate ca.cert for key ca.key
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x81161c3]

```
Clair and Postgresql images have been set up with the docker-compose file ( Clair/Coreos Github)
and are on the same subnet 172.22.x.x (network interface br-5ae2491b4bb1)

Additional informations:
Go version:
`go version go1.8 linux/amd64`


Best regards,

--- a/config/config.go
+++ b/config/config.go
@@ -86,6 +86,10 @@ func Init(cfgFile string, logLevel string, noClean bool) {
 	}
 	err := viper.ReadInConfig()
 	if err != nil {
+		e, ok := err.(viper.ConfigParseError)
+		if ok {
+			log.Fatalf("error parsing config file: %v", e)
+		}
 		log.Debugf("No config file used")
 	} else {
 		log.Debugf("Using config file: %v", viper.ConfigFileUsed())

